### PR TITLE
Closes #83 - Use Accumulo native maps by default

### LINKED ISSUE #83: Use Accumulo native maps by default
I checked. Native maps are disabled by default. They should be enabled, built, and used by default.

### PR DESCRIPTION
**Pull request by mikewalch**

### CODE REVIEW

File: bin/impl/setup.sh [hunk 1]
```diff
@@ -131,6 +131,7 @@ $SED "s#ACCUMULO_TSERVER_OPTS=.*#ACCUMULO_TSERVER_OPTS=\"-Xmx$ACCUMULO_TSERV_MEM
 $SED "s#ACCUMULO_DCACHE_SIZE#$ACCUMULO_DCACHE_SIZE#" "$ACCUMULO_HOME"/conf/accumulo-site.xml
 $SED "s#ACCUMULO_ICACHE_SIZE#$ACCUMULO_ICACHE_SIZE#" "$ACCUMULO_HOME"/conf/accumulo-site.xml
 $SED "s#ACCUMULO_IMAP_SIZE#$ACCUMULO_IMAP_SIZE#" "$ACCUMULO_HOME"/conf/accumulo-site.xml
+"$ACCUMULO_HOME"/bin/build_native_library.sh
```
Comment (keith-turner) [inline bin/impl/setup.sh]: Do you know what happens if this does not run successfully?

Comment (mikewalch) [inline bin/impl/setup.sh]: There is `set -e` before this command so the script will stop

**Comment by keith-turner** [comment]
May want to lower tserver heap if native maps are enabled, howerver trying to do this automatically is complicated.  Could make enabling native map a performance profile option.  If that was the case then when I enable native map I can chose to adjust tserver memory.


```
case "$PERFORMACE_PROFILE" in
  8GX2)
    #Enable Accumulo native map
    export ACCUMULO_USE_NATIVE_MAP=false
    #Accumulo tserver jvm heap size
    export ACCUMULO_TSERV_MEM=768m
    #Accumulo data cache size
    export ACCUMULO_DCACHE_SIZE=256M
    #Accumulo index cache size
    export ACCUMULO_ICACHE_SIZE=128M
    #Accumulo in mem map size
    export ACCUMULO_IMAP_SIZE=128M
    # Yarn Node Manager max memory (in MB)
    export YARN_NM_MEM_MB=8192
    # Yarn Node Manager max cpu vcores
    export YARN_NM_CPU_VCORES=8
    # Number of threads per worker process
    export FLUO_WORKER_THREADS=10
    # Worker process memory
    export FLUO_WORKER_MEM_MB=1024
    # Number worker processes
    export FLUO_WORKER_INSTANCES=1
    ;;

```

**Comment by mikewalch** [comment]
Made enabling native maps optional in 130064f9e51a0bf

**Comment by keith-turner** [comment]
Also need to modify the following prop in accumulo-site.xml

```xml
  <property>
    <name>tserver.memory.maps.native.enabled</name>
    <value>false</value>
  </property>
```

**Comment by mikewalch** [comment]
Modified prop in accumulo-site.xml in 16ab40c93a10d98c0d86603388fe3330e0b3659a

**Comment by keith-turner** [comment]
+1

**Pull request merged**